capabilities.go

```diff
@@ -30,6 +30,32 @@ func (c Capabilities) Browser(name string) Capabilities {
 	return c
 }
 
+// A ProxyConfig instance defines the desired proxy configuration the WebDriver
+// should use to proxy a Page.
+//
+//    ProxyType is required and defines the type of proxy being used
+//    Possible Values:
+//        {direct|manual|pac|autodetect|system}
+//
+// See: https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities#proxy-json-object
+type ProxyConfig struct {
+	ProxyType          string `json:"proxyType"`
+	ProxyAutoconfigUrl string `json:"proxyAutoconfigUrl,omitempty"`
```

Comment:
The rest of these need consistent casing as well: `Url` -> `URL`, `Http` -> `HTTP`, `Socks` -> `SOCKS`, etc.